e2e/stack_test.go

```diff
@@ -149,8 +224,12 @@ func TestE2E(t *testing.T) {
 			require.NoError(t, err)
 			require.Fail(t, fmt.Sprintf("expected tx to be deposit tx: %s", txBytes))
 		}
+		if len(txs) > 1 {
+			t.Logf("%d txs", len(txs))
+		}
 	}
 	t.Log("Monomer blocks contain the l1 attributes deposit tx")
+	// time.Sleep(15 * time.Second)
```

Comment:
Can you remove this comment before merge?

Reply:
Always sneaks back in. I just love sleeping for 15 seconds.